pkg/cmd/server/bootstrappolicy/controller_policy.go

```diff
@@ -54,7 +54,28 @@ func init() {
 			rbac.NewRule("create").Groups(buildGroup, legacyBuildGroup).Resources("builds/docker", "builds/source", "builds/custom", "builds/jenkinspipeline").RuleOrDie(),
 			rbac.NewRule("get").Groups(imageGroup, legacyImageGroup).Resources("imagestreams").RuleOrDie(),
 			rbac.NewRule("get", "list", "create", "delete").Groups(kapiGroup).Resources("pods").RuleOrDie(),
-			rbac.NewRule("get").Groups(kapiGroup).Resources("namespaces").RuleOrDie(),
+			eventsRule(),
+		},
+	})
+
+	// build-pod-controller
+	addControllerRole(rbac.ClusterRole{
```

Comment:
@csrwng need to make sure I don't miss any client calls you do and I don't have any extra listed here...